install/kubernetes/helm/subcharts/tracing/values.yaml

```diff
@@ -65,4 +65,7 @@ ingress:
     # - secretName: tracing-tls
     #   hosts:
     #     - tracing.local
-gatewayName: ingressgateway
+
+gateway:
+  enabled: false
+  name: ingressgateway
```

Comment:
why does Jaeger permit setting the ingress, while other addons don't?  Probably something to handle in a followon PR.

Reply:
I do not know - I wasn't the author of that :) This just moves the existing setting under the "gateway" section, as suggested by @objectiser.

Reply:
sounds fine.  thanks.